fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -3543,6 +3548,32 @@ private void createOlapTable(Database db, CreateTableStmt stmt) throws DdlExcept
         olapTable.setIndexSchemaInfo(baseIndexId, tableName, baseSchema, schemaVersion, schemaHash,
                 shortKeyColumnCount);
 
+
+        for (AlterClause alterClause : stmt.getRollupAlterClauseList()) {
+            AddRollupClause addRollupClause = (AddRollupClause)alterClause;
+
+            // TODO(wangbo): 2020/2/3 doris auto select base rollup
```

Comment:
Remove this comment.  One Olap table always has one base index.

Reply:
👌